components/Layout/Layout.tsx

```diff
@@ -1,16 +1,20 @@
-import { ErrorBanner, Header, Panel } from "components";
+import { ErrorBanner, Header } from "components";
 import { ReactNode } from "react";
+import styled from "styled-components";
 
 interface Props {
   children: ReactNode;
 }
 export function Layout({ children }: Props) {
   return (
-    <>
+    <Main>
       <ErrorBanner />
       <Header />
-      <main>{children}</main>
-      <Panel />
-    </>
+      {children}
+    </Main>
   );
 }
+
+const Main = styled.main`
```

Comment:
This is also required for doing `height:100%` to take up the full height of the view.